cli/azd/docs/featue-toggle.md

```diff
@@ -0,0 +1,61 @@
+# Feature Toggle
+
+The Azure Developer CLI includes a `feature toggle` which allows the cli to ship `in-preview` ( _a.k.a. experimental_ ). This document provides the feature design for it.
+
+## Main requirement
+
+As a customer, I can explicitly ask `azd` to unblock and show any _experimental_ functionality. The features becoming available are not finalized, so they can be changed or removed in future releases.
+
+The strategy to ask `azd` to `toggle` between modes is:
+
+> azd config set experimental on
+
+and
+
+> azd config set experimental off
+
+## Experimental features
+
+When toggling experimental mode `on`, the Azure Developer CLI includes the next type of features:
+
+1. Command or sub-command
+    - The command becomes available to be run just like any other command.
+    - Customers are **not** expected to know when a command or sub-command is experimental by just typing the command. This means:
+      - There are not suffix or prefix for a in-preview command.
+      - There is not sub-grouping command for all experimental commands.
```

Comment:
Reword for simplicity:

```suggestion
    - The command is designed exactly as how it would be released.
      - There is no suffix or prefix to the command
      - There is no `experimental` command group which experimental commands live under.
```